README.md

```diff
@@ -1,128 +1,108 @@
-print(output["value"])
-```
+    ```bash
+    pipx install griptape-cli
+    ```
+
+   You can also install a pre-release version from GitHub.
+    ```bash
+    git clone https://github.com/griptape-ai/griptape-cli.git
+    ```
+
+    ```bash
+    pipx install ./griptape-cli
+    ```
+2. Verify installation
+    ```bash
+    gt --help
+    ```
+
+## Skatepark
+You can use the CLI to spin up a local emulator of the Griptape Cloud Skatepark.
+Use the Skatepark emulator to develop, test, and validate that your program will operate as expected when deployed as a Griptape Cloud Manage Structure.
+
+1. Start by creating a new repository in your own Github account from the [Managed Structure Template](https://github.com/griptape-ai/managed-structure-template).
+    a. Make sure you're logged in to GitHub.
+    b. Go to the [Managed Structure Template repo](https://github.com/griptape-ai/managed-structure-template).
+    c. Select the *Use this template* drop-down.
+    d. Choose *Create a new repository*.
+    e. Provide a name for the new repository and (optionally) a description.
+    f. Press the *Create repository* button.
+    g. You now have a repository in your own GitHub account that is a copy of the Managed Structure Template for you to begin working with.
+2. Clone your newly-created repository to a directory in your local development environment so that you can begin authoring your own Griptape Managed Structure.
+3. Start the Skatepark emulator.
+    ```bash
+    gt skatepark start
+    ```
+4. In a separate terminal window, register a Structure from within the directory of the locally cloned repository.
+    ```bash
+    gt skatepark register --directory structure/ --main-file structure.py
+    ```
+    This will result in the ID of the registered Structure. It is important to make note of this ID as it will be used to distinguish which Structure you want to run. You can register any number of Structures.
+
+    The example client program uses the environment variable `GT_STRUCTURE_ID` to determine which Structure to run.
+    Set this environment variable to the Structure ID you registered in the previous step.
+    ```bash
+    export GT_STRUCTURE_ID={STRUCTURE_ID}
+    ```
+
+    Or you can register and set the environment variable in one step.
+    ```bash
+    export GT_STRUCTURE_ID=$(gt skatepark register --directory structure/ --main-file structure.py)
+    ```
+
+> [!IMPORTANT]
+> Structures registered with the emulator are not persisted across restarts. You will need to re-register the Structure each time you restart the emulator.
+5. Confirm that the Structure is registered.
+    ```bash
+    gt skatepark list
+    ```
+    You should see a list of registered Structures and the directories they point to, confirming that your Managed Structure was properly registered
+6. You can load environment variables into your Structure by creating an `.env` file in the directory of the Structure you registered. 
```

Comment:
PHILOSOPHICAL QUESTION OF GREAT MERIT: is it "an .env file" or "a .env file" because some people say "dot EE-EN-VEE" but others might say "env file" oh lord oh lord

Reply:
oh lord oh lord. I say "env file" personally.

Reply:
you never lived through the terror that was "ESS-ENN-EE-ESS" vs. "snes"